src/pages/home/index.jsx

```diff
@@ -90,11 +91,19 @@ export function App() {
 
       <main className={changeLayoutAllScreen()}>
         <nav className={style["nav-bar"]}>
-          <Input
-            onChange={(event) => {
-              setSearch(event.currentTarget.value);
-            }}
-          />
+          <div className={style["content-input"]}>
+            <div className={style["input-wrapper"]}>
+              <button className={style["button-search"]}>
+                <MagnifyingGlass size={16} />
+              </button>
+              <Input
+                borderNone={"yes"}
+                onChange={(event) => {
+                  setSearch(event.currentTarget.value);
+                }}
```

Comment:
here you can past all the props of the native input like a onChange, placeholder, type... because on the input componente we receive the ...rest props that is all the native input props